samples/basic/test/using.test.ts

```diff
@@ -1,4 +1,4 @@
-import { describe, expect, it } from 'vitest';
```

Comment:
Can you just revert all changes to samples?

Reply:
Yeah sure. Good idea! 😊